salt/orchestrate/aws/xpro_elb.sls

```diff
@@ -0,0 +1,149 @@
+{% set env_settings = salt.cp.get_file_str("salt://environment_settings.yml")|load_yaml %}
+{% set ENVIRONMENT = salt.environ.get('ENVIRONMENT', 'mitxpro-qa') %}
+{% set env_data = env_settings.environments[ENVIRONMENT] %}
+{% set PURPOSES = salt.environ.get('PURPOSES', 'xpro-qa').split(',') %}
+{% set VPC_NAME = env_data.vpc_name %}
+{% set BUSINESS_UNIT = salt.environ.get('BUSINESS_UNIT', env_data.business_unit) %}
+
+{% set launch_date = salt.status.time(format="%Y-%m-%d") %}
+{% set subnet_ids = salt.boto_vpc.describe_subnets(
+    vpc_id=salt.boto_vpc.describe_vpcs(
+        name=env_data.vpc_name).vpcs[0].id
+    ).subnets|map(attribute='id')|list %}
+{% set security_groups = salt.pillar.get('edx:lb_security_groups', ['default', 'edx-{env}'.format(env=ENVIRONMENT)]) %}
+{% set defined_purposes = env_data.purposes %}
+
+{% for purpose_name in PURPOSES %}
+{% set codename = defined_purposes[purpose_name].versions.codename %}
+{% set release_version = salt.sdb.get('sdb://consul/edxapp-{}-{}-release-version'.format(ENVIRONMENT, codename)) %}
+{% set purpose = defined_purposes[purpose_name] %}
+{% set domain_map = {
+'xpro-production':{
+          'cms': 'studio-xpro.mitx.mit.edu',
+          'lms': 'xpro.mitx.mit.edu',
+          'preview': 'preview-xpro.mitx.mit.edu'
+          },
+'xpro-qa': {
+          'cms': 'studio-xpro-qa.mitx.mit.edu',
+          'lms': 'xpro-qa.mitx.mit.edu',
+          'preview': 'preview-xpro-qa.mitx.mit.edu'
+  },
+'sandbox': {
+          'cms': 'studio-xpro-qa-sandbox.mitx.mit.edu',
+          'lms': 'xpro-qa-sandbox.mitx.mit.edu',
+          'preview': 'preview-xpro-qa-sandbox.mitx.mit.edu'
+  }
+} %}
```

Comment:
Why aren't we using the new hostnames in this `domain_map` and if we should be, wouldn't be best if we pulled those values from `environment_settings.yml`?